kinto_algolia/views.py

```diff
@@ -30,19 +29,22 @@ def search_view(request, **kwargs):
     bucket_id = request.matchdict['bucket_id']
     collection_id = request.matchdict['collection_id']
 
+    # algoliasearch doesn't support pagination
+    # https://github.com/algolia/algoliasearch-client-python/issues/365
```

Comment:
Is that still a thing?

Reply:
Yes I created https://github.com/Kinto/kinto-algolia/issues/4 so that we can investigate